<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cortarForm</class>
 <widget class="QWidget" name="cortarForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>671</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Largura Inicial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputcortarFormii">
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Largura Final</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputcortarFormifi">
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Altura Inicial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputcortarFormji">
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Altura Final</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputcortarFormjf">
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Esta função corta uma região da imagem. Defina os valores iniciais e finais de largura e altura para delimitar a região.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
